server/calendar/calendar-router.js

```diff
@@ -0,0 +1,77 @@
+var calendarController = require('./calendar-controller.js');
+var express = require('express');
+var calendarRouter = express.Router();
+var User = require('./User');
+
+
+calendarRouter.get('/', calendarController.signin);
+
+calendarRouter.get('/callback', calendarController.callback);
+
+calendarRouter.post('/checkloggedin', calendarController.isAuthenticated);
+
+calendarRouter.get('/logout', calendarController.logout);
+
+//Temporary Failure route if passport authentication fail
+calendarRouter.get('/failure', function() {
+  console.log('Something went wrong in authenticating with google');
+});
+
+//Temporary route to get all users calendar meta-information
+calendarRouter.get('/allCalendars', function (req, res) {
```

Comment:
Yeah thats why I commented these are all temporary. At the moment these functions exist just so I can test out the Google API. Doing it like this allows me to navigate to specific route to test them out rather than having to hard code calls from somewhere on the backend.  

Once we have the google account set up and the backend working properly the methods will all just exist in the controller.